system/system.proto

```diff
@@ -29,7 +29,7 @@ import "types/types.proto";
 
 option go_package = "github.com/openconfig/gnoi/system";
 
-option (types.gnoi_version) = "1.1.2";
+option (types.gnoi_version) = "1.1.3";
```

Comment:
Sorry, this needs to be `1.2.0` based on adding fields.

Thanks -- r.

Reply:
Fixed it. Thanks.